app/controllers/api/v1/emotions_controller.rb

```diff
@@ -39,8 +60,12 @@ def response_hash
   end
 
   def build_three_set
-    Emotion.positive.sample(NUMBER_OF_ELEMENTS) +
-      Emotion.neutral.sample(NUMBER_OF_ELEMENTS) +
-      Emotion.negative.sample(NUMBER_OF_ELEMENTS)
+    Emotion.where(public: true).positive.sample(NUMBER_OF_ELEMENTS) +
+      Emotion.where(public: true).neutral.sample(NUMBER_OF_ELEMENTS) +
+      Emotion.where(public: true).negative.sample(NUMBER_OF_ELEMENTS)
+  end
```

Comment:
@Maryna-Harasko please define in emotion model scope:
`scope :public, -> { where(public: true) } `
and use it here
```
 Emotion.public.positive.sample(NUMBER_OF_ELEMENTS)
 ...
```